smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/trait-value/extra-trait-property.errors

```diff
@@ -1 +1 @@
-[WARNING] smithy.example#MyInteger: Error validating trait `range`: Invalid structure member `maxx` found for `smithy.api#range` | TraitValue
+[WARNING] smithy.example#MyInteger: Error validating trait `range`: Invalid structure member `maxx` found for `smithy.api#range` | TraitValue.smithy.api#range.maxx
```

Comment:
Maybe we should update the hierarchical suppression matching to also treat `#` like `.` so you could suppress an entire namespace?

Reply:
I guess that would be smithy team's call. Trebuchet won't be doing any partial matching on these eventIds, only exact matching